test.ps1

```diff
@@ -0,0 +1,7 @@
+Remove-Item -Recurse .\tools\NUnit.Runners.*
+.\tools\nuget install NUnit.Runners -Version 2.6.4 -OutputDirectory tools
```

Comment:
Not particularly keen about maintaining a 3MB executable in the repo just for the test script. It would be equally effective to use `Invoke-WebRequest` to download a specific version if it's not already there, i.e.:

    Invoke-WebRequest https://dist.nuget.org/win-x86-commandline/v3.4.4/NuGet.exe -OutFile .\tools\nuget.exe

Re-installing NUnit on each run should be avoided. If one has a repo cloned and working offline then it becomes impossible to run tests. Tests should be runnable offline if they have been run at least once while online (and while it would be ideal to have an offline setup right after cloning, this is still better than not being able to run them at all).